supplier_app/tests/test_forms.py

```diff
@@ -22,7 +23,7 @@
 
 from supplier_app.tests import (
     taxpayer_creation_POST_factory,
-    get_bank_info_example
+    get_bank_info_example,
 )
```

Comment:
We should consider using isort for all dependencies

Reply:
Some people of the team has already implemented, I will try to implement it

Reply:
resolved